chapters/fundamentals.txt

```diff
@@ -1260,14 +1260,14 @@ application and implementation.
 [NOTE]
 .Note
 ====
-If ename:VK_ERROR_UNKNOWN is received, the application should be checked
+If ename:VK_ERROR_UNKNOWN is received, the application should: be checked
```

Comment:
Strictly speaking, notes don't need normative language.

Reply:
Right, in that case Notes should avoid keywords all the same, unless they are referencing normatively established requirement. It is still confusing and I think some Notes still raise real requirements.